src/main/scala/mesosphere/marathon/api/v2/PodNormalization.scala

```diff
@@ -42,10 +43,27 @@ object PodNormalization {
     }
   }
 
+  def normalizeScheduling(pod: Pod): Option[PodSchedulingPolicy] = {
```

Comment:
I am not sure what problem is this method solving ... maybe it deserves a docs or a better name?